addons/crm/models/crm_lead.py

```diff
@@ -161,8 +161,12 @@ class CrmLead(models.Model):
     date_conversion = fields.Datetime('Conversion Date', readonly=True)
     date_deadline = fields.Date('Expected Closing', help="Estimate of the date on which the opportunity will be won.")
     # Customer / contact
+    company_partner_id = fields.Many2one('res.partner', string='Company Partner', domain="[('is_company', '=', True)]")
```

Comment:
For me all that is garbage, would vote to not do it.

Reply:
:cry: but yes it's a lot of noise for a very tiny UX feature in the end.